### DISCUSSION TITLE
How named entity recognition result is used?

### DISCUSSION THREAD

**Discussion by hsiaoyi0504** [castorini/pyserini#904] [Q&A] [answered]
Although checking several documents, I still not sure how the named entity recognition result will be used like what's the default behavior if there is NER key in the collection JSON files?

**Comment by lintool** [comment, marked as answer]
Currently, NE information is simply stored in the index as part of the raw document if the `-storeRaw` option is used. So, there is no "default" behavior.

**Reply by hsiaoyi0504** [reply]
Thanks for answering. Originally, I thought there is some default mechanism that utilize the NER result to improve the searching.
I misunderstand it early anyway. Thanks again.
